velox/functions/sparksql/In.cpp

```diff
@@ -143,6 +156,7 @@ void registerIn(const std::string& prefix) {
   registerInFn<Varchar>(prefix);
   registerInFn<Timestamp>(prefix);
   registerInFn<Date>(prefix);
+  registerInFnForDecimal(prefix);
```

Comment:
Perhaps we don't need the 'registerInFnForDecimal' function and could just register them as below.

> registerInFn<ShortDecimal<P1, S1>>(prefix);
  registerInFn<LongDecimal<P1, S1>>(prefix);